Tribler/Test/Core/Modules/RestApi/test_create_torrent_endpoint.py

```diff
@@ -1,14 +1,14 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
Please don't use this encoding directive. Instead, you can use the `decode` method to convert a string to unicode so you don't have to write unicode characters in the source code.